src/app/app-routing.module.ts

```diff
@@ -1,7 +1,15 @@
 import { NgModule } from '@angular/core';
```

Comment:
Make compatible all imports.
I mean with spaces in brackets or without - decide.

Reply:
fixed